oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -103,15 +103,11 @@ static void mca_spml_ucx_waitall(void **reqs, int *count_p)
 
     SPML_VERBOSE(10, "waiting for %d disconnect requests", *count_p);
     for (i = 0; i < *count_p; ++i) {
-        do {
-            opal_progress();
-            status = ucp_request_test(reqs[i], NULL);
-        } while (status == UCS_INPROGRESS);
+        status = opal_common_ucx_wait_request(reqs[i], mca_spml_ucx.ucp_worker);
```

Comment:
shall we move mca_spml_ucx_waitall() to common as well?

Reply:
@yosefe will do after logging unification is complete